ast/dml.go

```diff
@@ -135,7 +135,29 @@ type TableName struct {
 
 // Restore implements Recoverable interface.
 func (n *TableName) Restore(sb *strings.Builder) error {
-	return errors.New("Not implemented")
+	//Joining indexHints.
+	indexHints := func(sb *strings.Builder) error {
+		for i, value := range n.IndexHints {
+			if i >= 0 {
```

Comment:
Remove the condition, `i >= 0` is always true.

Reply:
thank you